localstack/plugins.py

```diff
@@ -13,3 +13,14 @@ def configure_edge_port(container):
     for port in ports:
         if port:
             container.ports.add(port)
+
+
+# Register the PartitionAdjustingProxyListener only if the default region is a region in the AWS GovCloud partition
+@hooks.on_infra_start(should_load=lambda: config.DEFAULT_REGION.startswith("us-gov-"))
```

Comment:
Technically this should always be active since we support multi-region. Using DEFAULT_REGION is actually reviving a legacy feature in a way because this *should* be redundant now.

Reply:
Changed to use an explicit feature flag `PARTITION_ALIGNMENT` (which is of type truthy).